src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -133,7 +133,101 @@ const timezoneLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log)
         'in your logging configuration. For more details, see ' +
-        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.md'
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx'
+    );
+  }
+  return settings;
+};
+
+const destLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.dest')) {
+    log(
+      '"logging.dest" has been deprecated and will be removed ' +
+        'in 8.0. To set the destination moving forward, you can use the "console" appender ' +
+        'in your logging configuration or define a custom one. For more details, see ' +
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx.'
+    );
+  }
+  return settings;
+};
+
+const quietLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.quiet')) {
+    log(
+      '"logging.quiet" has been deprecated and will be removed ' +
+        'in 8.0. Moving forward, you can use "logging.root.level:error" in your logging configuration. '
+    );
+  }
+  return settings;
+};
+
+const silentLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.silent')) {
+    log(
+      '"logging.silent" has been deprecated and will be removed ' +
+        'in 8.0. Moving forward, you can use "logging.root.level:off" in your logging configuration. '
+    );
+  }
+  return settings;
+};
+
+const verboseLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.verbose')) {
+    log(
+      '"logging.verbose" has been deprecated and will be removed ' +
+        'in 8.0. Moving forward, you can use "logging.root.level:all" in your logging configuration. '
+    );
+  }
+  return settings;
+};
+
+const jsonLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.json') && settings.env !== 'development') {
+    log(
+      '"logging.json" has been deprecated and will be removed ' +
+        'in 8.0. To specify log message format moving forward, ' +
+        'you can configure the "appender.layout" property for every custom appender in your logging configuration. ' +
+        'There is currently no default layout for custom appenders and each one must be declared explicitly. ' +
+        'For more details, see ' +
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx.'
+    );
+  }
+  return settings;
+};
+
+const logRotateDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (
+    has(settings, 'logging.rotate.enabled') ||
+    has(settings, 'logging.rotate.usePolling') ||
+    has(settings, 'logging.rotate.pollingInterval') ||
+    has(settings, 'logging.rotate.everyBytes') ||
+    has(settings, 'logging.rotate.keepFiles')
+  ) {
```

Comment:
NIT: not 100% sure, but I think `has(settings, 'logging.rotate)` may be sufficient here.